diff --git a/node_modules/@react-native-material/core/lib/typescript/Dialog.d.ts b/node_modules/@react-native-material/core/lib/typescript/Dialog.d.ts
index 43dd902..447b050 100644
--- a/node_modules/@react-native-material/core/lib/typescript/Dialog.d.ts
+++ b/node_modules/@react-native-material/core/lib/typescript/Dialog.d.ts
@@ -2,6 +2,7 @@ import React from 'react';
 export interface DialogProps {
     visible?: boolean;
     onDismiss?: () => void;
+    children?:React.ReactNode
 }
 declare const Dialog: React.FC<DialogProps>;
 export default Dialog;
diff --git a/node_modules/@react-native-material/core/lib/typescript/base/Provider.d.ts b/node_modules/@react-native-material/core/lib/typescript/base/Provider.d.ts
index 9b42e66..dead0e2 100644
--- a/node_modules/@react-native-material/core/lib/typescript/base/Provider.d.ts
+++ b/node_modules/@react-native-material/core/lib/typescript/base/Provider.d.ts
@@ -3,5 +3,5 @@ import { SpacingFuncProviderProps } from 'react-native-flex-layout';
 import { ThemeProviderProps } from './ThemeContext';
 import { WindowSizeClassProviderProps } from './WindowSizeClassContext';
 import { IconComponentProviderProps } from './IconComponentContext';
-export declare type ProviderProps = ThemeProviderProps & WindowSizeClassProviderProps & SpacingFuncProviderProps & IconComponentProviderProps;
+export declare type ProviderProps = ThemeProviderProps & WindowSizeClassProviderProps & SpacingFuncProviderProps & IconComponentProviderProps &{children?:React.ReactNode};
 export declare const Provider: React.FC<ProviderProps>;
